src/components/ai-avatar/ai-avatar.tsx

```
@@ -0,0 +1,85 @@
+            </Host>
+        );
+    }
+
+    private getHostAriaLabel(): string {
+        let thinkingText = '';
+        if (this.isThinking) {
+            thinkingText =
+                ' (' + this.getTranslation('ai-avatar.thinking') + ')';
+        }
+
+        return 'AI Avatar' + thinkingText;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)